cea/inputlocator.py

```diff
@@ -367,6 +367,15 @@ def get_archetypes_schedules(self, region):
         to the scenario if they are not yet present, based on the configured region for the scenario."""
         return self._get_region_specific_db_file(region, 'archetypes', 'occupancy_schedules.xlsx')
 
+    def get_archetypes_system_controls(self, region):
+        """ Returns the database of region-specific system control parameters. These are copied
+        to the scenario if they are not yet present, based on the configured region for the scenario.
+
+        :param region:
+        :return:
+        """
+        return self._get_region_specific_db_file(region, 'archetypes', 'system_controls.xlsx')
```

Comment:
could we use a csv file instead? since there is only one worksheet anyway...

Reply:
potentially there could be more worksheets.
containing temperatures of comfort, control set points of systems, etc.
I'm just following whatever is already there.

Reply:
ok. thank you for explaining your reasoning. let's stick with it!